pkg/resources/crd.go

```diff
@@ -0,0 +1,124 @@
+package resources
+
+import (
+	"fmt"
+
+	arkv1 "github.com/heptio/ark/pkg/apis/ark/v1"
+
+	apiextv1beta1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1beta1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// ListCRDs returns a list of the CRD types for all of the required Ark CRDs
+func ListCRDs() []*apiextv1beta1.CustomResourceDefinition {
+	return []*apiextv1beta1.CustomResourceDefinition{
+		backupCRD(),
+		scheduleCRD(),
+		restoreCRD(),
+		configCRD(),
+		downloadRequestCRD(),
+		deleteBackupRequestsCRD(),
+	}
+}
+
+func backupCRD() *apiextv1beta1.CustomResourceDefinition {
+	return &apiextv1beta1.CustomResourceDefinition{
```

Comment:
Would you want to create a helper function that takes in a `resource string` and then returns a CRD? Everything is the same except for objectMeta.name and spec.names.plural & .kind. We could uppercase the first letter to get kind. WDYT?

Reply:
Yup, good idea.